README.md

```diff
@@ -9,12 +9,7 @@ The tool's API is unstable and might be a subject to a further change.
 
 ## Setup
 
-Clone the project.
-* On macOS, this is enough.
-* On Linux, after cloning go to `src/nativeInterop` directory and run the `setup.sh` script. 
```

Comment:
Why you removed these instructions?
Did you solve the problem and now the C files are compiled automatically by the build script?

Reply:
Yes! I did it without a PR some time ago